libs/utils/executor.py

```diff
@@ -272,13 +272,29 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        errors = False
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
+            return True
+        for name in tc['files']:
+            logging.info('%14s - Write value into file: \'%s\' -> \'%s\'',
+                         'Files', tc['files'][name], name)
+            try:
+                self.target.write_value(name, tc['files'][name], True)
+            except Exception:
+                logging.debug('%14s - Write value into file failed: \'%s\' -> \'%s\'',
```

Comment:
I think this should be `logging.error`. It's actually quite likely that errors will occur here, e.g. the if user is running the wrong kernel version/config, and we should complain loudly.